src/app/src/main/kotlin/org/vaccineimpact/orderlyweb/controllers/web/UserController.kt

```diff
@@ -19,5 +19,10 @@ class UserController(context: ActionContext,
                 .sortedBy { it.displayName.toLowerCase() }
     }
 
+    fun getUserEmails(): List<String>
+    {
+        return userRepo.getUserEmails()
```

Comment:
So we're assuming that we always have a manageable number of users to be able to return the full list on each call? No value in doing an initial filter with first few letters here? Probably not for the number we have in Montagu, would just make the UI less responsive.

Reply:
I think so, yeah. The number of users would have to be pretty high for this to be a problem.